repository.rosinstall

```diff
@@ -16,4 +16,4 @@
 - git:
     local-name: common_resources
     uri: https://github.com/shadow-robot/common_resources.git
-    version: kinetic-devel
+    version: F_adding_common_msgs
```

Comment:
have the changes required here being merged?

Reply:
So we still have it in a branch since the "grasp" msg was supposed to be moved back to the private version of common_resources. Should we figure that out before we merge this PR?

Reply:
I think we only reference to this msg: from sr_msgs_common.srv import MoveObject
and we should keep this in public common_resources

Reply:
Ok, changing to kinetic-devel, should be merged after this PR:
https://github.com/shadow-robot/common_resources/pull/22